src/test/java/support/matchers/RequestMatchers.java

```diff
@@ -0,0 +1,20 @@
+package support.matchers;
+
+import static support.matchers.JsonObjectMatchers.hasJsonPath;
+
+import org.hamcrest.Matcher;
+
+import io.vertx.core.json.JsonObject;
+
+public final class RequestMatchers {
+
```

Comment:
Personally, I wouldn't bother with the blank lines for between the class and the constructor